src/parser.ts

```diff
@@ -230,6 +230,7 @@ export function newParser() {
     SyncNewBoardState({ squaresWithLetters, playerScores }) {
       if (startIndex !== undefined && game.you.name === "") {
         identifyPlayerOne(squaresWithLetters);
+        getStartingLetters(squaresWithLetters);
```

Comment:
I think these two functions could be merged together into a function called something like `identifyPlayers`

Reply:
Had a version like this at first, but `identifyPlayerOne` has two paths it takes. I wanted to keep the code DRY.

Reply:
Oh right, my initial read made me think that there was more repetition this way around, but you're right - they're not as similar as I first thought